applications/rag/tests/ray_job.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import ray
+
+@ray.remote
+def hello_world():
+    return "hello world"
+
+# Automatically connect to the running Ray cluster.
+ray.init()
+print(ray.get(hello_world.remote()))
```

Comment:
fix new line here

Reply:
file deleted. ray job is submitted in-line from CLI command.